stanford_image_styles.features.inc

```diff
@@ -832,6 +832,21 @@ function stanford_image_styles_image_default_styles() {
     ),
   );
 
+  // Exported image style: square_370_x_370.
+  $styles['square_370_x_370'] = array(
```

Comment:
Perhaps you can add the hi-res option to this image style as well so we don't get grainy images.